source/assets/stylesheets/la.scss

```diff
@@ -110,16 +115,25 @@ h6 {
 }
 
 .hero-background {
-  background-image: url('/assets/images/la-hero-short.jpeg');
+  background-image: url("/assets/images/la-hero.jpg");
   background-position: top center;
-  /* background-size: cover; */
+  background-size: cover;
+}
+
+.hero__text {
+  line-height: 1.3;
+  margin-bottom: 1rem;
+  // Make the main text in the hero image not run over the palm tree.
+  @include tablet {
+    width: 70%;
```

Comment:
A fix for the text not being readable over the palm tree. Seems to work with all screen sizes.